### ISSUE TITLE
Support custom components

### ISSUE THREAD

**Issue by danestves** [payloadcms/remix-server#12]
As soon as I put custom components and `scss` to payload, I get the following error building and serving the `server`

![CleanShot 2023-02-09 at 12 45 37](https://user-images.githubusercontent.com/31737273/217881201-a6e8fb2f-28bc-4722-9f59-874f86807eec.png)

I searched a lot of what I need to change but until now I don't know how to do it 😅

**Comment by DanRibbens** [comment]
What do you have in your `tsconfig.json` for `compilerOptions.jsx`?

This is what `create-payload-app` gives you which should work:

```json
{
  "compilerOptions": {
    "target": "es5",
    "lib": [
      "dom",
      "dom.iterable",
      "esnext"
    ],
    "allowJs": true,
    "strict": false,
    "esModuleInterop": true,
    "skipLibCheck": true,
    "outDir": "./dist",
    "rootDir": "./src",
    "jsx": "react",
    "paths": {
      "payload/generated-types": [
        "./src/payload-types.ts",
      ],
    }
  },
  "include": [
    "src"
  ],
  "exclude": [
    "node_modules",
    "dist",
    "build",
  ],
  "ts-node": {
    "transpileOnly": true,
    "swc": true,
  }
}
```

**Comment by danestves** [comment]
This is when running the `pnpm build`

![CleanShot 2023-02-09 at 13 14 49](https://user-images.githubusercontent.com/31737273/217888282-12b5da62-b850-485b-8fd7-18637d3075bd.png)

This is the only blocker that I have with this template 😅 everything else work like a charm without custom blocks/fields for payload

**Comment by dsod** [comment]
The issue was that we re-exported our payload configuration file from `apps/cms/src/index.ts`, which caused the whole configuration file to be imported (since it has a default export) when importing anything from `apps/server/index.ts`.

Simply remove any imports of the payload configuration file from `apps/cms/src/index.ts` and custom components will work again.

Fixed the boilerplate in 118264963a30ce2d73a483622e50a5b0f7e254a8
